src/Components/AlbumCard/AlbumCard.css

```diff
@@ -0,0 +1,45 @@
+.card {
+  margin: 3%;
```

Comment:
I'm a fan of using <code>rem</code> units for elements that are primarily text based, you'll have to tell me your thoughts on that but instead of using percentages they manage margin and padding when screen width shrinks so small 3% would be hard to notice